R/splitByRanks.R

```diff
@@ -55,42 +63,29 @@
 #' changes to \code{rowData} of the base object are not returned, whereas only 
 #' the \code{colData} of the base object is kept. 
 #'
-#' @seealso
-#' \code{\link[=splitOn]{splitOn}}
-#' \code{\link[=unsplitOn]{unsplitOn}}
-#' \code{\link[=agglomerate-methods]{agglomerateByVariable}},
-#' \code{\link[scuttle:sumCountsAcrossFeatures]{sumCountsAcrossFeatures}},
-#' \code{\link[=agglomerate-methods]{agglomerateByRank}},
-#' \code{\link[SingleCellExperiment:altExps]{altExps}},
-#' \code{\link[SingleCellExperiment:splitAltExps]{splitAltExps}}
-#'
-#' @name splitByRanks
+#' @rdname agglomerate-methods
 #'
 #' @examples
 #' data(GlobalPatterns)
 #' # print the available taxonomic ranks
 #' taxonomyRanks(GlobalPatterns)
 #'
-#' # splitByRanks
-#' altExps(GlobalPatterns) <- splitByRanks(GlobalPatterns)
-#' altExps(GlobalPatterns)
-#' altExp(GlobalPatterns,"Kingdom")
-#' altExp(GlobalPatterns,"Species")
+#' # agglomerateByRanks
+#' # 
+#' x <- agglomerateByRanks(GlobalPatterns)
+#' altExps(x)
+#' altExp(x,"Kingdom")
+#' altExp(x,"Species")
 #'
 #' # unsplitByRanks
-#' x <- unsplitByRanks(GlobalPatterns)
+#' x <- unsplitByRanks(x)
 #' x
-NULL
-
-################################################################################
-# splitByRanks
-
-#' @rdname splitByRanks
+#'
```

Comment:
Add NULL

Reply:
Here I added the agglomerateByRanks information to agglomerate-methods man page. If I add NULL, I think it does not add all the information written above to the man page and only add agglomerateByRanks methods to Usage part

Reply:
If you add #' @rdname agglomerate-methods to line 83 it should work. If not, then this is ok. This NULL resets roxygen settings --> that way we have full control on where these document chunks are going.

Reply:
I tried but it did not work, I am not sure but NULL may work as intended only with document chunks using #' @name xyz

Reply:
Might be. Thanks!